src/components/ItaliaTheme/manage/Widgets/CharCounterTextareaWidget.jsx

```diff
@@ -42,19 +42,17 @@ const CharCounterTextareaWidget = (props) => {
       <TextArea
         id={`field-${id}`}
         name={id}
-        value={value?.slice(0, 255) || ''}
+        value={value?.slice(0, forceMaxLength ? maxLength : 5000) || ''}
         disabled={props.isDisabled}
         placeholder={placeholder}
         onChange={({ target }) => {
           setTextChar(target.value);
           onChange(
             id,
-            target.value === '' ? undefined : target.value.slice(0, 255),
+            target.value === '' ? undefined : target.value.slice(0, forceMaxLength ? maxLength : 5000),
```

Comment:
```suggestion
            target.value === '' ? undefined : forceMaxLength?target.value.slice(0, maxLength):target.value,
```

Reply:
non mi è chiaro questo suggerimento di modifica... c'era qualcosa di sbagliato in quella che ho proposto, che questo sistema?